libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/intersectionUtils.ts

```diff
@@ -42,16 +42,28 @@ export function convertIntersectionToAttributeFilters(
             const ref = h.attributeHeader.ref;
             if (backendSupportsElementUris || enableDuplicatedLabelValuesInAttributeFilter) {
                 result.push({
-                    attributeFilter: newPositiveAttributeFilter(ref, {
-                        uris: [h.attributeHeaderItem.uri],
-                    }),
-                    primaryLabel: h.attributeHeader.primaryLabel,
+                    attributeFilter: {
```

Comment:
Can these be moved to some helper function, since is repeated on [line 59](https://github.com/gooddata/gooddata-ui-sdk/pull/5140/files#diff-c5130f23012fbacb4044eaa806b38458e97d4b7d3bffd079a88805add7ed2381R59)